src/Products/PluginIndexes/DateRangeIndex/DateRangeIndex.py

```diff
@@ -252,33 +250,22 @@ def _apply_index(self, request, resultset=None):
         used.
         """
         iid = self.id
-        record = parseIndexRequest(request, iid, self.query_options)
+        record = parseIndexRequest(request, self.id, self.query_options)
```

Comment:
If you aren't going to use `iid` assigned above, delete the assignment too (it isn't used afterward).